CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.14.4 (Next)
 
 * [#289](https://github.com/slack-ruby/slack-ruby-client/pull/289): Fix reconnects when ping timers under/overshoot - [@georgyangelov](https://github.com/georgyangelov).
+* [#291](https://github.com/slack-ruby/slack-ruby-client/pull/291): Update README concerning OAuth and legacy tokens - [@jcraigk](https://github.com/jcraigk).
```

Comment:
No need for a CHANGELOG for docs, ignore danger complaining.

Reply:
Just curious - why keep a manual CHANGELOG instead of auto-generating release notes from the commits?